enable-tls-between-clients-and-servers: what variable to check (#21843)
pingcap/docs: enable-tls-between-clients-and-servers.md

6 rows in set (0.0062 sec)
```

> A non-empty `Ssl_cipher` value indicates that your connection is secure.

Reviewer: ```suggestion
If the `Ssl_cipher` value is not empty, the connection is encrypted.
```
